docs/faq/general/what-languages-can-be-used-with-metabase.md

```diff
@@ -22,6 +22,7 @@ The languages you can currently pick from in Metabase are:
 - Polish
 - Portuguese
 - Russian
+- Serbian
```

Comment:
was this an intentional change in this PR?

Reply:
Yes. Thought I'd fix it up while I was in there.